jest.config.base.js

```diff
@@ -11,8 +11,14 @@ module.exports = {
     "^(unmock|unmock-cli|unmock-core|unmock-jsdom|unmock-node)(?:/dist)?((?:/.*)|$)":
       "<rootDir>/../../packages/$1/src$2"
   },
-  setupFilesAfterEnv: ["<rootDir>/../../jest.setup.js"],
   clearMocks: true,
+  transformIgnorePatterns: [
+      "/node_modules/(?!loas3)",
```

Comment:
Required in Node 6 as loas does not transpile and uses stuff like `Object.entries`.